test/func/fixtures/db-migrations/failed-describe.testplane.js

```diff
@@ -0,0 +1,39 @@
+describe('failed describe', function() {
+    it('successfully passed test', async ({browser}) => {
+        await browser.url(browser.options.baseUrl);
+
+        assert.isTrue(true);
+    });
+
+    it('test without screenshot', async ({browser}) => {
+        await browser.url(browser.options.baseUrl);
+
+        await browser.assertView('header', 'header');
+    });
+
+    it('test with image comparison diff', async ({browser}) => {
+        await browser.url(browser.options.baseUrl);
+
+        await browser.assertView('header', 'header');
+    });
+
+    it('test with long error message', async () => {
+        throw new Error(`long_error_message ${'0123456789'.repeat(20)}\n message content`);
+    });
+
+    it('test with successful assertView and error', async ({browser}) => {
+        await browser.url(browser.options.baseUrl);
+
+        await browser.assertView('header', 'header');
```

Comment:
If it is "successful assertView" lets add `{ignoreDiffPixelCount: '100%'}` as 3rd argument to make it 100% stable

Reply:
These are never intended to run, report should be fixed, but just in case, added that option.